metricbeat/docs/modules/istio.asciidoc

```diff
@@ -28,10 +28,19 @@ in <<configuration-metricbeat>>. Here is an example configuration:
 [source,yaml]
 ----
 metricbeat.modules:
+# Istio mesh. To collect all all Mixer-generated metrics
 - module: istio
   metricsets: ["mesh"]
   period: 10s
+  # use istio-telemetry.istio-system:42422, when deploying Metricbeat in a kuberentes cluster
   hosts: ["localhost:42422"]
+
+# Istio mixer. To monitor Mixer itself.
+- module: istio
+  metricsets: ["mixer"]
+  period: 10s
+  # use istio-telemetry.istio-system:15014, when deploying Metricbeat in a kuberentes cluster
```

Comment:
Is this service exposed in `localhost:15014` in the host network? Then this line might be confusing, because when we deploy Metricbeat in a cluster we recommend to do it in the host network, and then `localhost:15014` would work too.

Reply:
hmm, maybe the comment was not so clear. What I want to say is "when deploying Metricbeat in a kubernetes cluster as pod". thanks for mentioning, will fix this.